streams/streams-scala/src/main/scala/org/apache/kafka/streams/scala/kstream/Consumed.scala

```diff
@@ -43,7 +43,7 @@ object Consumed {
     ConsumedJ.`with`(keySerde, valueSerde, timestampExtractor, resetPolicy)
 
   /**
-   * Create an instance of [[Consumed]] with key and value [[Serde]]s.
+   * Create an instance of [[Consumed]] with key and value Serdes.
```

Comment:
Should this be `[[Serdes]]` -- Would need to resolve the "conflict" between `org.apache.kafka.streams.scala.Serdes` (deprecated; we should not use) and `org.apache.kafka.streams.scala.serialization.Serdes` though.

Similar elsewhere.

Reply:
What is meant here is that we are talking about two `Serde` instances, key and value, and not the `Serdes` factory class, or not?